### ISSUE TITLE
Member files have appropriate view operations

### ISSUE THREAD

**Issue by KristinaTaylor** [PTC-ALM/TortoiseSI#35]
Single selection member files only have:
View History
View Annotated Revision
View Member Differences
View Member Information

These should all use the wf execute command.

**Comment by bleskows** [comment]
Note for when we verify. Use an client build of 7876 or later. This build contains a fix for a defect where wf execute threw an out of bounds error for single character options (i.e. -g).

**Comment by bleskows** [comment]
![image](https://cloud.githubusercontent.com/assets/10633761/6528590/f997e67e-c3f0-11e4-8c0f-9d5fb8fcf9bd.png)
@KristinaTaylor do you think the word "View" makes these commands too wordy? **Edit:** Although I guess this is what we do in the eclipse integration, so best to keep it consistent.

Also would you have any suggestions for icons for viewhistory and memberinfo?
TortoiseGit doesn't exactly provide similar commands for these ops. You'd have to look in the git repo to see what I'm talking about since I can't drop in ico files, but I was thinking something like this:
View history: src/Resources/menurevisiongraph.ico
View member inf: src/Resources/menuproperties.ico

![image](https://cloud.githubusercontent.com/assets/10633761/6529341/04d04090-c3f6-11e4-92a8-cbfa7df5f0a1.png)

If you spot something better, I could use that too.

**Comment by KristinaTaylor** [comment]
I think you've got the right choices here. You can drop the "View" if you add a separator that puts space between actions and views. I'm thinking we organize it similar to the git menu:

![gitmenu](https://cloud.githubusercontent.com/assets/10520385/6530992/6335641e-c403-11e4-8718-bb79b8a1364b.png)

Top is "resync" type of actions, next diff, then the views, then the actions, and near the bottom is the exclude filter. Spacers in between each group.

**Comment by bleskows** [comment]
![image](https://cloud.githubusercontent.com/assets/10633761/6531699/fb0a7c84-c408-11e4-8541-b06f0fafb40e.png)
So with the currently available actions, I can provide something like this. We'll try to follow the "resync" actions, diff, views, member actions + exclude structure for future issues (i.e. #18 )

**Comment by KristinaTaylor** [comment]
:+1:

**Comment by bleskows** [comment]
I will wait before creating a pull request on this one. In #59 , Andrej is making some changes to the rc files for the resource strings, and it's probably easier for me to merge in my changes after that is pushed to master than the other way around.

**Comment by bleskows** [comment]
Verify with build [1.0.83.0](https://ci.appveyor.com/project/PTC-ALM/tortoisesi/build/1.0.83.0). Use client 7876 or later.

**Comment by sharonzhao** [comment]
Verified using client build of 7887 and TortoiseSI build of 1.0.85.0.
The following view operations display on the right-click menu of a single member file:
- Member Differences
- Member History
- Annotated Revision
- Member Information

**Comment by bleskows** [comment]
Removed needs docs: documented in Alpha release User Guide
